### PR TITLE
[MIG] mgmtsystem_nonconformity_project: Migration to 12.0

### PR DESCRIPTION
**Pull request by luiseevaquer**

### CODE REVIEW

**Review by moylop260 — APPROVED** [review]
👍

**Review by hugho-ad — APPROVED** [review]

**Comment by moylop260** [comment]
Hello my friend @ivantodorovich 
Could you check it, please?

File: mgmtsystem_nonconformity_project/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+{
+    "name": "Management System - Project",
+    "version": "12.0.1.0.0",
+    "author": "Savoir-faire Linux,Odoo Community Association (OCA)",
+    "website": "http://www.savoirfairelinux.com",
```
Comment (ivantodorovich) [inline mgmtsystem_nonconformity_project/__manifest__.py]:
```suggestion
    "website": "http://github.com/OCA/management-system",
```

File: mgmtsystem_nonconformity_project/models/mgmtsystem_nonconformity_project.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+#
+##############################################################################
+
+from odoo import fields, models
+
+
+class MgmtsystemAction(models.Model):
+    _inherit = "mgmtsystem.action"
+
+    def _complete_name(self):
```
Comment (ivantodorovich) [inline mgmtsystem_nonconformity_project/models/mgmtsystem_nonconformity_project.py]:
```suggestion
    def _compute_complete_name(self):
```

https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#37models

> Method conventions
Compute Field: the compute method pattern is _compute_<field_name>
Inverse method: the inverse method pattern is _inverse_<field_name>
Search method: the search method pattern is _search_<field_name>
Default method: the default method pattern is _default_<field_name>
Onchange method: the onchange method pattern is _onchange_<field_name>
Constraint method: the constraint method pattern is _check_<constraint_name>

File: mgmtsystem_nonconformity_project/README.rst [hunk 3]
```diff
@@ -0,0 +1,10 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===========================================
+Management System - Nonconformity - Project
+===========================================
+
+This module enables you to set a project as an action in the nonconformity.
+
```
Comment (ivantodorovich) [inline mgmtsystem_nonconformity_project/README.rst:10]: Can you transform the readme to the new OCA format: splitted in chunks in a `readme` folder?

https://github.com/OCA/maintainer-tools/tree/master/template/module/readme

**Review by ivantodorovich — CHANGES_REQUESTED** [review]
Thanks for the migration @lescobarvx 
A few minor comments :+1: 

Also, there are many administrative and translations commits, can you squash them to clean the commit history?

https://github.com/OCA/maintainer-tools/wiki/Merge-commits-in-pull-requests

**Comment by luiseevaquer** [comment]
Hi @ivantodorovich 
I have already made the suggested changes, Could you check it again, please?

**Review by ivantodorovich — APPROVED** [review]

**Comment by ivantodorovich** [comment]
Thanks @lescobarvx !

/ocabot merge nobump

**Pull request merged**